recipes/nco/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+    name: nco
+    version: 4.5.5
+
+source:
+    git_url: https://github.com/nco/nco.git
+    git_tag: 4.5.5
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - gsl
+        - antlr
+        - curl
+        - zlib
+        - hdf5
+        - libnetcdf
+        - udunits2
+        - gcc
```

Comment:
Interesting one. I haven't used that before. Is this needed because the standard gcc doesn't work?